addons/binding/org.openhab.binding.tadoac/.project

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>org.openhab.binding.tadoac</name>
```

Comment:
I quite like tadoac. Bindings should not for a company, but for a certain kind of product/protocol. So if the tado products do not actually share any technical details, each should have its own binding. We e.g. also have binings like sonytv and epsonprojector.